app/controllers/idv/enter_password_controller.rb

```diff
@@ -129,7 +129,13 @@ def confirm_current_password
     def init_profile
       idv_session.create_profile_from_applicant_with_password(
         password,
-        resolved_authn_context_result.enhanced_ipp?,
+        is_enhanced_ipp: resolved_authn_context_result.enhanced_ipp?,
+        proofing_components: ProofingComponents.new(
```

Comment:
suggestion (non-blocking): Extract ProofingComponents creation to a private method.

(failed to submit with the review, adding now)